tdrs-backend/tdpservice/users/test/test_permissions.py

```diff
@@ -0,0 +1,79 @@
+"""Test appropriate permissions are assigned to each Group."""
+import pytest
+
+
+@pytest.mark.django_db
+def test_ofa_admin_permissions(ofa_admin):
+    """Test that an OFA Admin user inherits the correct permissions."""
+    expected_permissions = {
+        'admin.view_logentry',
+        'auth.view_group',
+        'data_files.add_datafile',
+        'data_files.view_datafile',
+        'stts.view_region',
+        'stts.view_stt',
+        'users.add_user',
+        'users.change_user',
+        'users.view_user',
+    }
+    group_permissions = ofa_admin.get_group_permissions()
+    assert group_permissions == expected_permissions
+
+
+@pytest.mark.django_db
+def test_ofa_system_admin_permissions(ofa_system_admin):
+    """Test that an OFA System Admin user inherits the correct permissions."""
+    expected_permissions = {
+        'admin.add_logentry',
+        'admin.change_logentry',
+        'admin.view_logentry',
+        'admin_interface.add_theme',
+        'admin_interface.change_theme',
+        'admin_interface.view_theme',
+        'auth.add_group',
+        'auth.add_permission',
+        'auth.change_group',
+        'auth.change_permission',
+        'auth.view_group',
+        'auth.view_permission',
+        'authtoken.add_token',
```

Comment:
Note we don't register some of these models in the admin so some of these won't show up in Django Admin even though the permissions is present. If we want to change that we just need to register the additional models.

Specifically the unregistered models are:
* authtoken.token
* authtoken.token_proxy
* contenttypes.contenttype
* core.globalpermission (entirely unused)
* sessions.session